src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random();
```

Comment:
Creating a new instance of 'Random' each time the method is called can be inefficient if the method is called multiple times. Consider making 'random' a class-level instance variable.

Reply:
fix comment